### ISSUE TITLE
Update language on 508 color contrast

### ISSUE THREAD

**Issue by jenniferthibault** [18F/brand#53]
See https://github.com/18F/web-design-standards/issues/884

See more in my chat with Nick Bristow and [the accessibility guild on slack](https://18f.slack.com/archives/g-accessibility/p1452619723000058).

**Comment by jenniferthibault** [comment]
**Our current text:**
> This matrix shows how to use the 18F brand colors for text and backgrounds in combinations that conform with section 508 of the Americans with Disabilities Act. This makes sure that there is a high  color contrast between the text color and the background color.

> Key: Please don't use these color combinations. They are not 508 conformant, which means that some people would have difficulty reading the text.

**Proposed change:**
> This matrix shows how to use the 18F brand colors for text and backgrounds in combinations that conform with the standards of Section 508 of the Rehabilitation Act. For more on color contrast, read [18F's Accessibility Guide](https://pages.18f.gov/accessibility/color/). Each of the combinations shown below meets or exceeds a contrast ratio of 4.5:1, and is approved for body text and headline text.

> Key: Please don't use these color combinations, they do not meet a color contrast ratio of 4.5:1, so they do not conform with the standards of Section 508 for body text. This means that some people would have difficulty reading the text. Employing accessibility best practices improves the user experience for all users.

Thoughts & edits welcome – @amberwreed @ericronne @nickbristow

**Comment by ericronne** [comment]
well put

**Comment by amberwreed** [comment]
Sounds good.

On Thu, Jan 14, 2016 at 7:41 AM, Eric Ronne <notifications@github.com>
wrote:

> well put
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/brand/issues/53#issuecomment-171677074>.
>



-- 
-------------------------
*Amber Reed*
Experience Designer
18F

**Comment by nickbristow** [comment]
love it @jenniferthibault
